aws/resource_aws_iam_group_policy_test.go

```diff
@@ -142,6 +174,28 @@ func testAccCheckIAMGroupPolicy(
 			return err
 		}
 
+		*groupPolicy = *output
+
+		return nil
+	}
+}
+
+func testAccCheckAWSIAMGroupPolicyNameChanged(i, j *iam.GetGroupPolicyOutput) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if aws.StringValue(i.PolicyName) == aws.StringValue(j.PolicyName) {
+			return errors.New("IAM Group Policy name did not change")
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckAWSIAMGroupPolicyNameMatches(i, j *iam.GetGroupPolicyOutput) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if aws.StringValue(i.PolicyName) != aws.StringValue(j.PolicyName) {
+			return errors.New("IAM Group Policy name did not match")
+		}
```

Comment:
both of these ignore casings, but that's probably fine

Reply:
I'm pretty sure in terms of the acceptance testing we can allow it to be more strict, but if it presents some sort of problem we can certainly implement the `strings.ToLower()` calls 👍